### PR TITLE
Update more cluster access privileges

### PR DESCRIPTION
**Pull request by mlepage-google**
#### Problem
Issue #14419 more clusters need custom required privilege

#### Change overview
- Add access elements to cluster XML to match spec
- Rename commissioning.xml to network-commissioning-cluster.xml
- Rename thermostat "ac compressor" attribute to "ac compressor type"

#### Testing
- ZAP regen, build, CI
- Review generated access.h

### CODE REVIEW

**Review by tcarmelveilleux — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml [hunk 1]
```diff
@@ -45,29 +53,81 @@ limitations under the License.
     <attribute side="server" code="0x0016" define="MAX_HEAT_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0BB8" optional="true">max heat setpoint limit</attribute>
     <attribute side="server" code="0x0017" define="MIN_COOL_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0640" optional="true">min cool setpoint limit</attribute>
     <attribute side="server" code="0x0018" define="MAX_COOL_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0C80" optional="true">max cool setpoint limit</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml:55]: The 4 min/max setpoint limit attributes need to have "manage" for write, no?

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml [hunk 2]
```diff
@@ -45,29 +53,81 @@ limitations under the License.
...
+      <access op="read" privilege="view"/>
+      <access op="write" privilege="manage"/>
+    </attribute>
+    <attribute side="server" code="0x0045" define="AC_LOUVER_POSITION" type="ENUM8" min="0x00" max="0x05" writable="true" default="0x00" optional="true">
+      <description>ac louver position</description>
+      <access op="read" privilege="view"/>
+      <access op="write" privilege="manage"/>
+    </attribute>
     <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="INT16S" min="0x954D" max="0x7FFF" writable="false" default="0x8000" optional="true">ac coil temperature</attribute>
     <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="ENUM8" min="0x00" max="0xFF" writable="true" default="0x00" optional="true">ac capacity format</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml:132]: ACCapacityFormat needs "manage" for write.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml (hunk 1 above)
Comment (mlepage-google) [reply src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml]: OK merged in master again, went over the file again, should be good now.

File: src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml (hunk 2 above)
Comment (mlepage-google) [reply src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml]: Merged in master, went over file again, should be good now.

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**